### ISSUE TITLE
Possible BEDLAM2 test set leak in CameraHMR preprocessed labels

### ISSUE THREAD

**Issue by hugorodet-poly** [pixelite1201/BEDLAM#69]
Hello, 

I am currently using the SMPL-X labels from the provided 6fps preprocessed CameraHMR labels, and I have noticed some issues that might indicate test set leak.

1) Additional renderjobs in CameraHMR
I have used the download scripts to get the 66 render jobs as mentioned in the paper. However, there are a total of 68 render jobs referenced in the CameraHMR with the two extra being:
- 20240805_5-10_250_busstation_orbit_zoom (there is a scene among the 66 render jobs with the same parameters but it's dated to 20241001)
- 20240621_1_250_archmodelsvol8_tracking (similarly, one among the 66 is dated to 20241107 instead)
For those two additional render jobs, no mp4 files are provided and I had to discard them.

2) Missing sequences from CameraHMR labels
The CameraHMR labels contain only 23623 sequences if I only consider the "official" 66 render jobs where I have the MP4 files, less than the 27480 mentioned in the BEDLAM2 paper.

3) Missing motions names from the BEDLAM2 -> AMASS CSV file.
On the BEDLAM2 website you provide a CSV file which maps every "body" name to its original motion sampled from AMASS. However, I found that there were a total of 1741 unique bodies not found in the CSV (e.g. us_1602_2XL_2102). Due to the different naming convention of MOYO bodies, I could easily tell which of those bodies were from MOYO and extrapolate the original AMASS motion name from the body name, and deduce that there were sampled from 17 unique AMASS sequences. 

As a result: 
- The BEDLAM2 paper claims that there are total of 171 motions from MOYO :  154 in the training set, with the remaining 17 MOYO motions (and labels) left in the holdout test set. 
- The bodies in the CameraHMR preprocessed labels use a total of 171 different MOYO motions, 154 of which were found in the BEDLAM2->AMASS file, and the remaining 17 which were not found (but we still have both the SMPL-X labels and the images).

This does point towards possible label leakage from the official leaderboard test set to the CameraHMR labels. The missing two render jobs mentioned in the CameraHMR labels reinforce my guess that the CameraHMR and official BEDLAM2 splits were made at different times and are not exactly consistent with each other. I am also guessing that the reason many bodies were not found in the BEDLAM2->AMASS file is by design because they are meant for the holdout test set, and not because they are missing by mistake.

I am trying to make sense of those observations and am concerned about a possible test set leak.

Thank you for your time
